ci.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eu
+
+main() {
+
+    which python
+    export PATH="/usr/local/opt/python/libexec/bin:$PATH"
+    which python
+
+    setup_pip
+    install_requirements
+    build_libs
+    run_tests
+}
+
+setup_pip() {
+    python -m pip install --upgrade pip
+}
+
+install_requirements() {
+    pip install -r requirements.txt
+    pip install numpy --upgrade
```

Comment:
@pedrocamargo ... why are you upgrading numpy here?
requirements.txt already has it included right?

Reply:
It should have the latest